idseq_dag/steps/generate_alignment_viz.py

```diff
@@ -304,12 +304,13 @@ def get_sequences_by_accession_list_from_s3(accession_id_groups,
         mutex = TraceLock("get_sequences_by_accession_list_from_s3", threading.RLock())
         nt_bucket, nt_key = nt_s3_path[5:].split("/", 1)
         for accession_id, accession_info in accession_id_groups.items():
+            entry = nt_loc_dict.get(accession_id)
```

Comment:
Dict lookup moves to parent instead of 64 children, because sqlite spec does not permit concurrent reads over the same connection.

Reply:
Btw, it would be quite easy to implement a connection pool for the 64 threads here if this turns out to be a performance bottleneck.